docs/components/contacts.rst

```diff
@@ -4,8 +4,170 @@ Contacts
 There's several ways to extend Contacts in Mautic.
 One of them is to show custom events in a Contact's event timeline - this document shows you how.
 
+.. vale off
+
+.. note:: In Mautic 1.4, Leads got renamed to Contacts. However, much of the code still refers to Contacts as Leads. 
+
+Creating Contacts
+*****************
+
+.. vale on
+
+To create a new Contact, use the ``\Mautic\LeadBundle\Entity\Lead`` entity. Review the code sample.
+
+.. code-block:: php
+
+  <?php
+  // plugins/HelloWorldBundle/Services/ContactService.php
+
+  declare(strict_types=1);
+
+  namespace MauticPlugin\HelloWorldBundle\Services;
+
+  use Mautic\CoreBundle\Helper\IpLookupHelper;
+  use Mautic\LeadBundle\Entity\Lead;
+  use Mautic\LeadBundle\Entity\LeadRepository;
+  use Mautic\LeadBundle\Model\FieldModel;
+  use Mautic\LeadBundle\Model\LeadModel;
+  use Mautic\LeadBundle\Tracker\ContactTracker;
+
+  class ContactService
+  {
```

Comment:
The [code block in the old docs](https://developer.mautic.org/#manipulating-contacts-(leads)) was severely outdated and referencing methods that no longer work. The code block below doesn't contain any non-working code, though it does reference some deprecated methods like ``getUniqueIdentiferFields()``. This should be "good enough for now".